test/corpus/element.txt

```diff
@@ -63,6 +63,32 @@ Named Blocks
 <:header></:header>
 ---
 
+(template
+  (element_node
+    (element_node_start
+      (tag_name))
+    (element_node_end
+      (tag_name))))
+
+==================
+Argument components
+==================
+<@item></@item>
+---
+
+(template
+  (element_node
+    (element_node_start
+      (tag_name))
+    (element_node_end
+      (tag_name))))
+
+==================
+Argument components
+==================
```

Comment:
Could one of these two `Argument components` test names be changed so they're unique?

Reply:
done! thanks!